### PR TITLE
chore(j-s): Use generated CaseCustodyRestrictions type

### PR DESCRIPTION
**Pull request by oddsson**
# Use generated CaseCustodyRestrictions type

[Asana](https://app.asana.com/0/1199153462262248/1205359398381996/f)

## What

Use generated CaseCustodyRestrictions type

## Why

We are moving towards using generated GraphQL types in the front end

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: apps/judicial-system/web/src/types/index.ts [hunk 1]
```diff
@@ -238,6 +239,7 @@ export interface TempCase
     | 'appealedByRole'
     | 'appealRulingDecision'
     | 'defendants'
+    | 'CaseCustodyRestrictions'
```
Comment (unakb) [inline apps/judicial-system/web/src/types/index.ts:242]: Should this not be requestedCustodyRestrictions?

File: apps/judicial-system/web/src/types/index.ts [hunk 2]
```diff
@@ -252,6 +254,7 @@ export interface TempCase
   appealedByRole?: UserRole
   appealRulingDecision?: CaseAppealRulingDecision
   defendants?: Defendant[]
+  custodyRestrictions: CaseCustodyRestrictions[]
```
Comment (unakb) [inline apps/judicial-system/web/src/types/index.ts:257]: and this as well I think

File: apps/judicial-system/web/src/types/index.ts (hunk 1 above)
Comment (oddsson) [reply apps/judicial-system/web/src/types/index.ts]: Yeb! Thanks, fixed

**Review by unakb — APPROVED** [review]
Nice 👍

**Pull request merged**